scripts.py

```diff
@@ -3,12 +3,17 @@
 import os
 import re
 from datetime import datetime
+from nose.tools import set_trace
```

Comment:
Mostly reorganizing the imports here. The only changes added were in core.lane, core.util, and controller.